debian/changelog

```diff
@@ -1,3 +1,17 @@
+ubuntu-advantage-tools (27.8~22.04.1) jammy; urgency=medium
+
+  * New upstream release 27.8 (LP: #1969125)
+    - entitlements: apply overrides from the contract response
+    - fips:
+      + unhold fips package when enabling fips-updates
+      + make fips service incompatible with fips-updates
```

Comment:
This makes sense to me, but may be confusing to others. Consider:
```suggestion
      + Automatically disable fips service before enabling fips-updates
```

Reply:
Good point, updating